partner/app/main.py

```diff
@@ -89,6 +104,21 @@ async def read_tilesp(
     results: int = Query(1, example=2),
 ):
     """Endpoint for requests from Contile."""
+    unknown_params = []
+    for param in request.query_params:
+        if param not in ACCEPTED_QUERY_PARAMS:
+            unknown_params.append(param)
+
+    if unknown_params:
+        return JSONResponse(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            content={"unaccepted query parameter": unknown_params},
+        )
+    logger.error(
```

Comment:
This log message must only be sent if we encounter unknown query parameters and before we return from this function.